### ISSUE TITLE
Unable to Import From Google Keep

### ISSUE THREAD

**Issue by eitwi** [streetwriters/notesnook-importer#103]
I have done everything as told in import guide. When finished, Notesnook gives description "Import successful. 0 errors occured" yet none of my notes appear. I've done it 2 times and the results are the same. Hope there is a solution.

**Comment by thecodrr** [comment]
@eitwi this happens after you get the .zip file from Notesnook Importer?

**Comment by eitwi** [comment]
Wait, do I should get any file from the Notesnook Importer? or do you mean after I get the .zip file from Google Keep?

Anyway, I import .zip to Notesnook Importer, everything run well, it says "Import successful. 0 errors occured", but none of my Google Keep notes appear in Notesnook.

**Comment by smikwily** [comment]
@eitwi Yes. You are supposed to use https://importer.notesnook.com/ to convert the files to NotesNook, then import the files it generates into NotesNook.

**Comment by thecodrr** [comment]
Duplicate of #64. Closing to consolidate discussion — please follow #64 for updates on Google Keep import issue.
